### PR TITLE
[Improvement](group commit) FE should throw NotFoundException if database or table not exist when loading data

### PR DESCRIPTION
**Pull request by hust-hhb**
## Proposed changes
If database or table not exist, fe should process NotFoundException and send error status code TStatusCode.NOT_FOUND to be.

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by mymeiyi — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java [hunk 1]
```diff
@@ -1083,6 +1083,10 @@ public TLoadTxnBeginResult loadTxnBegin(TLoadTxnBeginRequest request) throws TEx
             status.setStatusCode(TStatusCode.LABEL_ALREADY_EXISTS);
             status.addToErrorMsgs(e.getMessage());
             result.setJobStatus(e.getJobStatus());
+        } catch (MetaNotFoundException e) {
+            LOG.warn("failed to begin: {}", e.getMessage());
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java:1087]: not log e.getMessage, log e please, then we could get the real exception stack.

**Comment by hust-hhb** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**